ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -461,6 +493,7 @@ export default class ConfirmTransactionBase extends Component {
           networkName={networkName}
           type={txData.type}
           isBuyableChain={isBuyableChain}
+          onClickBlockaidSupport={onClickBlockaidSupport}
```

Comment:
[nit] I think we can move the `onClickBlockaidSupport` inside of TransactionAlerts. TransactionAlerts seems to only be used for ConfirmTransactionBase